Makefile

```diff
@@ -199,3 +199,24 @@ adr-gen:
 	@echo "--> Downloading ADR template"
 	@curl -sSL https://raw.githubusercontent.com/celestiaorg/.github/main/adr-template.md > docs/architecture/adr-template.md
 .PHONY: adr-gen
+
+## goreleaser: list goreleaser options and checks if goreleaser is installed
+goreleaser: Makefile
+	@echo " Choose a goreleaser command to run:"
+	@sed -n 's/^## goreleaser/goreleaser/p' $< | column -t -s ':' |  sed -e 's/^/ /'
+	@goreleaser --version
+.PHONY: goreleaser
+
+## goreleaser-build: builds the celestia-appd binary using goreleaser for your
+# local OS
```

Comment:
[nit] if this overflow lines, it won't show up in `make help` output. See:

<img width="874" alt="Screenshot 2023-09-07 at 11 04 47 PM" src="https://github.com/celestiaorg/celestia-app/assets/3699047/069421fb-40d8-4d7d-9c71-673c777fa681">

[nit] capitalize the first word and period at the end for consistency with other help commands.

```suggestion
## goreleaser-build: Builds the celestia-appd binary using GoReleaser for your local OS.
```